usrp3/tools/scripts/uhd_image_builder_gui.py

```diff
@@ -395,17 +395,12 @@ def file_dialog(self):
         Opens a dialog window to add manually the Out-of-tree module blocks
         """
         append_directory = []
-        filename = QtWidgets.QFileDialog.getOpenFileName(self, 'Open File', '/home/')[0]
```

Comment:
@ejk43 removing this line would also mean the the button that opens the dialog has to be removed. Also "populate_list" is not used by changing this chunk of code. Please see my comment below for implications.